inception-project/inception
# #1256 Integrate curation into annotation page

**What's in the PR**
The user should be able to open the curation tab and select the users whose annotations she wants to see. The annotations are shown in a differing color additionally to the users' annotations.
The user can accept other users' annotations into her own annotation document or the curation document.
An automatic merge of consistent annotations analogous to the existing implementation in the curation can be triggered by button click.

Implementation is done using an AnnotationEditorExtension to save the accepted annotations and trigger the render of the ones from other users.

**How to test manually**
* Check out following webanno branch https://github.com/webanno/webanno/pull/1618
* Set up a project and annotate some documents with different users
* Go to the annotation page and open the curation tab.
* Select users whose annotations you want to see
* Accept some of the shown annotations into your curated document

**Automatic testing**

**Documentation**
